pyformance/reporters/influx.py

```diff
@@ -56,12 +56,14 @@ def __init__(
             autocreate_database=False,
             clock=None,
             global_tags=None,
-            reporting_precision = ReportingPrecision.SECONDS
+            reporting_precision = ReportingPrecision.SECONDS,
+            rp=None
     ):
         """
         :param reporting_precision: The precision in which the reporter reports to influx.
         The default is seconds. This is a tradeoff between precision and performance. More
         coarse precision may result in significant improvements in compression and vice versa.
+        :param rp: The name of the retention policy of your database, InluxDB rp default is autogen.
```

Comment:
```suggestion
        :param rp: The name of the retention policy of your database, InluxDB rp default is "autogen".
```

Reply:
Also, if the default retention policy is "autogen" you can use that string as the default value of `rp`. This will eliminate the `if` below.